views/js/runner/provider/qti.js

```diff
@@ -273,10 +299,7 @@ define([
 
                 submit()
                  .then(updateStats)
-                 .then(computeNextMove)
-                 .catch(function(err){
-                    self.trigger('error', err);
-                 });
+                 .then(computeNextMove);
```

Comment:
Ok, but in this case there will be several `error` triggering, unless i keep the `catch()` on each points and remove the centralized error handling.